public/scss/components/Map.module.scss

```
@@ -10,17 +10,29 @@
 
 .mapSelectArea
 {
-  @extend .btn_transparent;
-  @include SmallSemiBold;
-  height: 103px;
+  @include flex(row, center, flex-start);
+  padding: 28px 18px 10px 18px !important;

```

```suggestion
  padding: 28px 18px 10px !important;
```

dibikin begini juga bisa